packages/react-celo/src/constants.tsx

```diff
@@ -251,16 +252,15 @@ export const PROVIDERS: {
           return false;
       }
     },
-  },
+  } as WalletConnectProvider,
   [SupportedProviders.CoinbaseWallet]: {
     name: SupportedProviders.CoinbaseWallet,
     type: WalletTypes.CoinbaseWallet,
     description: 'Scan a QR code to connect your wallet',
     icon: CoinbaseWallet,
     canConnect: () => true,
     showInList: () => true,
-    listPriority: () => Priorities.Popular,
-    supportedPlatforms: [Platform.Mobile, Platform.Web],
```

Comment:
intentional?

Reply:
`supportedPlatforms` is only for walletconnect providers